frontend/src/components/Forms/Doctor/DoctorHelpers.tsx

```diff
@@ -22,17 +22,17 @@ const languagesSpokenOptions = [
 
 
 
-//FORM VALIDATOR
+//DOCTOR FORM VALIDATOR
 export const validationSchema = Yup.object().shape({
     drGivenName: Yup.string()
         .required('Given name(s) cannot be blank')
         .matches(/^[A-Za-z ]+$/, 'Only alphabets are allowed for this field'),
     drSurname: Yup.string()
         .required('Surname cannot be blank')
         .matches(/^[A-Za-z ]+$/, 'Only alphabets are allowed for this field'),
-    drPreferredName: Yup.string().matches(
-        /^[A-Za-z ]+$/,
-        'Only alphabets are allowed for this field'
+    drPreferredName: Yup.string()
+        .required('Preffered Name cannot be blank')
```

Comment:
```suggestion
        .required('Preferred Name cannot be blank')
```
Just made a little typo here that needs to be fixed before merging ☺️